contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,112 @@
+---
+date: 2022-08-29
+title: 'Array 1.39.0: Moving person properties onto events'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.39.0 introduces a new beta option which could improve query speed by up to 400x. And that's just for starters!
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.39.0 release notes
+
+**Release highlights:**
+
+- Beta: [Person properties on events](#beta-person-properties-have-moved-on-to-events)
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Apps: JOE to add
+
+### Beta: Person properties on events
+
+We're getting ready to make a substantial change to the way [persons](/manual/persons) and [events](/manual/events) work by essentially combining them and moving all persons properties _onto_ events. 1.39.0 gives you the chance to try the feature ahead of the full release by toggling your `actors_on_events_teams` [instance setting](/docs/self-host/configure/instance-settings).
+
+So, why have we moved person properties onto events? In part it's because it streamlines how we store data as we continue to scale, but it's also to unlock significant speed increases on queries involving both persons and events. Previously, PostHog stored personIDs and event information in separate tables, but this update simplifie
+
+A big benefit of this approach is that it enables significantly faster queries because we no longer have to compare multiple tables when running queries - everything is stored on the event. Some of our internal tests have shown a 400x increase in speed!
+
+We'll share more about this shift once it's fully released to all users - keep and eye on [the PostHog blog](/blog) for updates. 
```

Comment:
I'd add something about how `filtering by unique users` and `filtering by person properties` will work differently once enabled - read more about how it all works and what changed + how to enable this on your self-hosted instance

Reply:
Yep, just waiting on a link from @pjhul before I add this in.